### PR TITLE
Correct markdown presentation

### PR DESCRIPTION
**Pull request by cafferata**
| Q             | A
| ------------- | ---
| Branch        | master for features and deprecations
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Documented?   | yes
| Fixed tickets | 

# New Task Checklist:

- [x] Is the README.md file updated?
- [x] Are the dependencies added to the composer.json suggestions?
- [x] Is the doc/tasks.md file updated?
- [x] Are the task parameters documented?
- [x] Is the task registered in the tasks.yml file?
- [x] Does the task contains phpspec tests?
- [x] Is the configuration having logical allowed types?
- [x] Does the task run in the correct context?
- [x] Is the `run()` method readable?
- [x] Is the `run()` method using the configuration correctly?
- [x] Are all CI services returning green?

### CODE REVIEW

**Comment by cafferata** [comment]
See also the online Github rich diff:

1. [doc/tasks/composer_require_checker.md](https://github.com/phpro/grumphp/pull/552/files?short_path=41f73d4#diff-41f73d4d4da2223033def6e8fdfbc575)
1. [doc/tasks/phpstan.md](https://github.com/phpro/grumphp/pull/552/files?short_path=81da76f#diff-81da76f26e723874953fbcd4a99e0c43)
1. [doc/tasks/psalm.md](https://github.com/phpro/grumphp/pull/552/files?short_path=f509ff4#diff-f509ff422f9512252f2078f808add0db)

File: doc/tasks/composer_require_checker.md [hunk 1]
```diff
@@ -23,13 +23,13 @@ parameters:
 
 **composer_file**
 
-*Default: ~*
+*Default: \~*
```
Comment (Landerstraeten) [inline doc/tasks/composer_require_checker.md]:
```suggestion
*Default: null*
```

File: doc/tasks/composer_require_checker.md [hunk 2]
```diff
@@ -23,13 +23,13 @@ parameters:
...
 
 The composer.json of your code base that should be checked.
 
 **config_file**
 
-*Default: ~*
+*Default: \~*
```
Comment (Landerstraeten) [inline doc/tasks/composer_require_checker.md]:
```suggestion
*Default: null*
```

File: doc/tasks/phpstan.md [hunk 3]
```diff
@@ -24,13 +24,13 @@ parameters:
 
 **autoload_file**
 
-*Default: ~*
+*Default: \~*
```
Comment (Landerstraeten) [inline doc/tasks/phpstan.md]:
```suggestion
*Default: null*
```

File: doc/tasks/phpstan.md [hunk 4]
```diff
@@ -24,13 +24,13 @@ parameters:
...
 
 With this parameter you can specify the path your project's additional autoload file path.
 
 **configuration**
 
-*Default: ~*
+*Default: \~*
```
Comment (Landerstraeten) [inline doc/tasks/phpstan.md]:
```suggestion
*Default: null*
```

File: doc/tasks/psalm.md [hunk 5]
```diff
@@ -25,7 +25,7 @@ parameters:
 
 **config**
 
-*Default: ~*
+*Default: \~*
```
Comment (Landerstraeten) [inline doc/tasks/psalm.md]:
```suggestion
*Default: null*
```

File: doc/tasks/psalm.md [hunk 6]
```diff
@@ -48,7 +48,7 @@ With this parameter you can run Psalm without using the cache file.
 
 **report**
 
-*Default: ~*
+*Default: \~*
```
Comment (Landerstraeten) [inline doc/tasks/psalm.md]:
```suggestion
*Default: null*
```

**Comment by cafferata** [comment]
>*Did you check all the .md files?*

Yep, with an regex find/replace by PHPStorm.

**Comment by Landerstraeten** [comment]
Thanks @cafferata!

**Pull request merged**